HW05.Rmd

```diff
@@ -109,17 +109,26 @@ CTRP_df_q1 <- CTRP_df %>%
   filter(cpd_name == "vorinostat") %>%
   group_by(cancer_type) %>%
   summarize(AUC = mean(area_under_curve)) %>%
+  drop_na(cancer_type) %>% #There is a NA cancer type, it's not the lowest value, so I eliminated it.
   arrange(AUC)
 
-CTRP_df_q1
-
-ggplot(CTRP_df_q1, aes(cancer_type, AUC)) +
-  geom_col() +
-  coord_flip()
+ggplot(CTRP_df_q1, aes(fct_inorder(cancer_type), AUC)) +
+  geom_col(fill = "turquoise4") +
```

Comment:
I know you peer-reviewed my HW so you saw it yourself, but: I found boxplots to be more informative for this because it shows clearly the mean/SD/outliers. You could try to add those lines yourself to a barplot, but that's extra work and looks less precise